nixos/modules/programs/npm.nix

```diff
@@ -14,9 +14,9 @@ in
       enable = mkEnableOption "<command>npm</command> global config";
 
       package = mkOption {
-        type = types.path;
+        type = types.path or null;
         description = "The npm package version / flavor to use";
-        default = pkgs.nodePackages.npm;
+        default = if hasAttr "npm" pkgs.nodePackages then pkgs.nodePackages.npm else null;
```

Comment:
Why can't we just install some sort of `nodejs` package here? Upstream recommends to use a `npm` bundled within a `nodejs`-package.

cc @marsam @doronbehar

Reply:
Did like you said, as @doronbehar suggested the same fix in the comments on PR.